chainbridge/src/lib.rs

```diff
@@ -220,35 +308,21 @@ decl_module! {
             Self::vote_for(who, nonce, src_id, call)
         }
 
-        /// Votes against the proposal IFF it exists.
-        /// (Note: Proposal cancellation not yet fully implemented)
+        /// Commits a vote against a provided proposal.
+        ///
+        /// This is not yet fully implemented.
+        ///
+        /// # <weight>
+        /// - Fixed, since execution of proposal should not be included
+        /// # </weight>
+        #[weight = SimpleDispatchInfo::FixedNormal(500_000)]
         pub fn reject(origin, nonce: DepositNonce, src_id: ChainId, call: Box<<T as Trait>::Proposal>) -> DispatchResult {
```

Comment:
Since the call does not matter, can we just trasfer the call hash instead? That would make the call bounded and save bandwidth.

Reply:
That makes sense. Will require some changes to how we are storing the proposals, going to consider lower priority for now. Opened #39